image/image.go

```diff
@@ -253,3 +237,52 @@ func NewImageAPIResponse(resp *http.Response, uri string) (e *ErrInvalidImageAPI
 	}
 	return
 }
+
+func addCollectionIDHeader(r *http.Request, collectionID string) {
+	if len(collectionID) > 0 {
+		r.Header.Add(common.CollectionIDHeaderKey, collectionID)
+	}
+}
+
+// doGetWithAuthHeaders executes clienter.Do setting the user and service authentication token as a request header. Returns the http.Response and any error.
```

Comment:
This is the only method here that has a comment. Also the comment mentions the auth tokens but not the collection header. I suggest either completing the other methods/headers or else deciding to remove the comment so there's consistency.